relm4/Cargo.toml

```diff
@@ -42,6 +42,7 @@ adw = { version = "0.6", optional = true, package = "libadwaita" }
 flume = "0.11.0"
 futures = "0.3.30"
 fragile = "2.0.0"
+gio = "0.19"
```

Comment:
Including `gio` is not necessary because it is re-exported under `gtk::gio`: https://gtk-rs.org/gtk4-rs/git/docs/gtk4/index.html#reexports

Reply:
Oh, I missed that, thanks. Fixed now.